test/test_lpf.py

```diff
@@ -0,0 +1,44 @@
+import os
+import unittest
+
+from compression import lpf
+from generator import rand
+
+
+class TestLPF(unittest.TestCase):
+  run_large = unittest.skipUnless(
+    os.environ.get('LARGE', False), 'Skip test in small runs')
+
+  def check_input(self, text, answer=None):
```

Comment:
Nit: I think I assumed in all other tests that answers are always provided. Please check other test_* files and try to conform to that notation.

Reply:
Ack